### PR TITLE
r/ami_from_instance - tag on create

### PR DESCRIPTION
**Pull request by DrFaust92**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Output from acceptance testing:

```
$ make testacc TESTARGS='-run=TestAccAWSAMIFromInstance_'
--- PASS: TestAccAWSAMIFromInstance_basic (308.11s)
--- PASS: TestAccAWSAMIFromInstance_tags (393.96s)
--- PASS: TestAccAWSAMIFromInstance_disappears (422.17s)
```

### CODE REVIEW

**Comment by ewbankkit** [comment]
[Announcement](https://aws.amazon.com/about-aws/whats-new/2020/12/amazon-machine-images-support-tag-on-create-tag-based-access-control/).

Note:

> This capability is available at no additional cost in all AWS Regions, except AWS GovCloud (US) and AWS China Regions.

**Comment by ewbankkit** [comment]
Verified acceptance tests:

```console
$ make testacc TEST=./aws/ TESTARGS='-run=TestAccAWSAMIFromInstance_' ACCTEST_PARALLELISM=2
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./aws -v -count 1 -parallel 2 -run=TestAccAWSAMIFromInstance_ -timeout 120m
=== RUN   TestAccAWSAMIFromInstance_basic
=== PAUSE TestAccAWSAMIFromInstance_basic
=== RUN   TestAccAWSAMIFromInstance_tags
=== PAUSE TestAccAWSAMIFromInstance_tags
=== RUN   TestAccAWSAMIFromInstance_disappears
=== PAUSE TestAccAWSAMIFromInstance_disappears
=== CONT  TestAccAWSAMIFromInstance_basic
=== CONT  TestAccAWSAMIFromInstance_disappears
--- PASS: TestAccAWSAMIFromInstance_disappears (251.95s)
=== CONT  TestAccAWSAMIFromInstance_tags
--- PASS: TestAccAWSAMIFromInstance_basic (274.48s)
--- PASS: TestAccAWSAMIFromInstance_tags (309.80s)
PASS
ok  	github.com/terraform-providers/terraform-provider-aws/aws	561.859s
```

**Review by bateller — APPROVED** [review]
Looking forward to this change

File: aws/resource_aws_ami_from_instance_test.go [hunk 1]
```diff
@@ -80,6 +81,28 @@ func TestAccAWSAMIFromInstance_tags(t *testing.T) {
...
 
+func TestAccAWSAMIFromInstance_disappears(t *testing.T) {
+	var image ec2.Image
+	rName := acctest.RandomWithPrefix("tf-acc-test")
+	resourceName := "aws_ami_from_instance.test"
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckAWSAMIFromInstanceDestroy,
```
Comment (bflad) [inline aws/resource_aws_ami_from_instance_test.go:92]:
```suggestion
		PreCheck:     func() { testAccPreCheck(t) },
		ErrorCheck:   testAccErrorCheck(t, ec2.EndpointsID),
		Providers:    testAccProviders,
		CheckDestroy: testAccCheckAWSAMIFromInstanceDestroy,
```

**Review by bflad — APPROVED** [review]
Looks great, thank you, @DrFaust92 🚀 

Output from acceptance testing in AWS Commercial:

```
--- PASS: TestAccAWSAMIFromInstance_disappears (259.64s)
--- PASS: TestAccAWSAMIFromInstance_basic (276.96s)
--- PASS: TestAccAWSAMIFromInstance_tags (309.87s)
```

Output from acceptance testing in AWS GovCloud (US):

```
--- PASS: TestAccAWSAMIFromInstance_disappears (289.51s)
--- PASS: TestAccAWSAMIFromInstance_basic (307.13s)
--- PASS: TestAccAWSAMIFromInstance_tags (356.92s)
```

**Pull request merged**